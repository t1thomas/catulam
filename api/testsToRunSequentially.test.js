const dataMutations = require('./tests/dataMutations.js');
// const userMutations = require('./tests/userMutations.js');

describe('sequentially run tests', () => {
  dataMutations();
  // userMutations();
});
