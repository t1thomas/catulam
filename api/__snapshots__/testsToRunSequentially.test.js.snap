// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`sequentially run tests Mutations Add project data 1`] = `
Object {
  "AddSprintProject": Object {
    "from": Object {
      "id": "sp1test",
    },
    "to": Object {
      "id": "p1test",
    },
  },
  "AddTicketProject": Object {
    "from": Object {
      "id": "t1test",
    },
    "to": Object {
      "id": "p1test",
    },
  },
  "AddUserStoryProject": Object {
    "from": Object {
      "id": "us1test",
    },
    "to": Object {
      "id": "p1test",
    },
  },
}
`;

exports[`sequentially run tests Mutations Create 3 Sprints 1`] = `
Object {
  "sprint1": Object {
    "id": "sp1test",
    "sprintNo": 1,
  },
  "sprint2": Object {
    "id": "sp2test",
    "sprintNo": 2,
  },
  "sprint3": Object {
    "id": "sp3test",
    "sprintNo": 3,
  },
}
`;

exports[`sequentially run tests Mutations Create 3 UserStories 1`] = `
Object {
  "uStory1": Object {
    "id": "us1test",
    "issueNumber": 4,
    "storyText": "As a User I want to be able to first see a dashboard on my landing page",
  },
  "uStory2": Object {
    "id": "us2test",
    "issueNumber": 5,
    "storyText": "As a user I want to securely login to the app using my credentials",
  },
  "uStory3": Object {
    "id": "us3test",
    "issueNumber": 6,
    "storyText": "As a user I my profile data to be shown in my dashboard",
  },
}
`;

exports[`sequentially run tests Mutations Create 3 projects 1`] = `
Object {
  "prog1": Object {
    "desc": "First test project",
    "id": "p1test",
    "label": "TP1",
    "title": "Test Project 1",
  },
  "prog2": Object {
    "desc": "Second test project",
    "id": "p2test",
    "label": "TP2",
    "title": "Test Project 2",
  },
  "prog3": Object {
    "desc": "Third test project",
    "id": "p3test",
    "label": "TP3",
    "title": "Test Project 3",
  },
}
`;

exports[`sequentially run tests Mutations Create 3 tickets 1`] = `
Object {
  "tick1": Object {
    "desc": "Create API for backend and frontend",
    "done": false,
    "hourEstimate": 5,
    "id": "t1test",
    "issueNumber": 1,
    "title": "Set Up API",
  },
  "tick2": Object {
    "desc": "Create Login Page for users to enter credentials",
    "done": false,
    "hourEstimate": 5,
    "id": "t2test",
    "issueNumber": 2,
    "title": "login page",
  },
  "tick3": Object {
    "desc": "Create landing page for client side SPA",
    "done": false,
    "hourEstimate": 5,
    "id": "t3test",
    "issueNumber": 3,
    "title": "Landing page",
  },
}
`;

exports[`sequentially run tests Mutations Create a project 1`] = `
Object {
  "data": Object {
    "CreateProject": null,
  },
  "errors": Array [
    [GraphQLError: Cannot read property 'cookies' of undefined],
  ],
  "extensions": undefined,
  "http": Object {
    "headers": Headers {
      Symbol(map): Object {},
    },
  },
}
`;

exports[`sequentially run tests Mutations Link 3 Tickets to Sprints 1`] = `
Object {
  "SprintTic1": Object {
    "from": Object {
      "id": "t1test",
    },
    "to": Object {
      "id": "sp1test",
    },
  },
  "SprintTic2": Object {
    "from": Object {
      "id": "t2test",
    },
    "to": Object {
      "id": "sp2test",
    },
  },
  "SprintTic3": Object {
    "from": Object {
      "id": "t3test",
    },
    "to": Object {
      "id": "sp3test",
    },
  },
}
`;

exports[`sequentially run tests Mutations Link 3 Tickets to UserStories 1`] = `
Object {
  "UStoryTic1": Object {
    "from": Object {
      "id": "t1test",
    },
    "to": Object {
      "id": "us1test",
    },
  },
  "UStoryTic2": Object {
    "from": Object {
      "id": "t2test",
    },
    "to": Object {
      "id": "us2test",
    },
  },
  "UStoryTic3": Object {
    "from": Object {
      "id": "t3test",
    },
    "to": Object {
      "id": "us3test",
    },
  },
}
`;

exports[`sequentially run tests User Login Mutations Add user to project 1`] = `
Object {
  "AddUserProjects": Object {
    "from": Object {
      "id": "user1test",
    },
    "to": Object {
      "id": "p1test",
    },
  },
}
`;
